docker-entrypoint.sh-rootless

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
For the name of the file, maybe docker_entrypoint_rootless.sh ?
Let's keep the shell extension.